specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -70,6 +70,16 @@ Explain the user problem that need to be solved.
 
 ## High-level design
 
+#### Folder Structure
+
+* [frontend](../../projects/frontend-ui) The root folder for all frontend code.
+    * [shared-components](../../projects/frontend-ui/shared-components/README.md): the root folder for the shared components project, for more details please see the readme.
+      * gui: This contains the root package.json file for the project.
+        * projects:
```

Comment:
@sabadzhiev Here is the suggested project structure. 

changes:
1. `frontend-ui` to `frontend`
2. `shared` to `shared-components`


I think we should delete the `frontend/shared-components/gui/projects` folder and have documentation-ui and shared live in the gui folder directly. 
What do you think?

Reply:
@ivakoleva @tozka please give your opinions.